app/views/terms-and-conditions/private-beta-1.njk

```diff
@@ -0,0 +1,271 @@
+      <li>conduct a review of the general health and welfare of the relevant Single Species; </li>
+      <li>provide you with the relevant test results from the tests described in condition 4.1, including the detailed laboratory analysis; </li>
+      <li>provide you with a written report (the "Report") and you must ask the Vet to follow the Phase 1 guidance setting out what should be in the Report;</li>
+      <li>provide you with a separate review summary sheet (the "Review Summary Sheet") as described in condition 5 to accompany the Report referred to in condition 3.4(e).</li>
+    </ol>
+
+    <h2 class="govuk-heading-m">4. Testing a Single Species </h2>
+    <p class="govuk-body">4.1 You must ask the Vet to carry out one (1) of the following tests of the relevant minimum number of the relevant Single Species on the date or dates the Vet carries out the Review: </h3>
+    <ol class="govuk-list--alphabet govuk-body">
+      <li>test at least six (6) beef cattle for Bovine Viral Diarrhoea (BVD) using a blood (serum) antibody test;</li>
+      <li>test any number of dairy cattle for BVD using an antibody bulk milk test or a polymerase chain reaction (PCR) bulk milk test or a blood (serum) antibody test;</li>
+      <li>test at least ten (10) sheep for the effectiveness of worming treatments containing one (1) of the following active chemicals:</li>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>Benzimidazole (BZ)</li>
+        <li>Levamisole (LV)</li>
+        <li>Macrocyclic lactones (ML)</li>
+        <li>Amino-acetonitrile derivatives (AD) </li>
+        <li>Spiroindoles (SI) </li>
+      </ul>
+      <li>test at least thirty (30) pigs for Porcine Reproductive and Respiratory Syndrome (PRRS) using an enzyme-linked immunosorbent assay (ELISA) or PRRS PCR test.</li>
+    </ol>
+    <p class="govuk-body">4.2 The tests set out in condition 4.1 are in addition to any other services which you may agree for the Vet to provide at the same time as the Review, but which are not covered by the Payment.</p>
+    <p class="govuk-body">4.3 You must ask the Vet to send the samples from the tests set out in condition 4.1 to a suitably accredited laboratory for the relevant analysis, the Vet guidance has further information on what a suitably accredited laboratory for each Single Species will be. </p>
+    <p class="govuk-body">4.4 You must ask the Vet to share the results analysis received from the laboratory explained in condition 4.3 for the relevant tests set out in condition 4.1.</p>
+    <p class="govuk-body">4.5 You cannot use test results you have already obtained under another accreditation scheme or any other legal or contractual obligation you are subject to in order to meet the testing requirements set out in condition 4.1.</p>
+
+    <h2 class="govuk-heading-m">5. Review Summary Sheet</h2>
+    <h3 class="govuk-heading-s">5.1	The Review Summary Sheet referred to in condition 3.4(f) must include all of the following: </h3>
```

Comment:
This is be a paragraph not a heading, using the heading class displays the text in bold